pvlib/ivtools.py

```diff
@@ -262,6 +264,177 @@ def fit_sde_sandia(voltage, current, v_oc=None, i_sc=None, v_mp_i_mp=None,
                                      v_oc)
 
 
+def fit_sdm_desoto(celltype, v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
+                   cells_in_series, temp_ref=25, irrad_ref=1000):
+    """
+    Calculates the five parameters for the single diode equation using
+    the De Soto et al. procedure described in [1]. This procedure has the
+    advantage of using common specifications given by manufacturers in the
+    datasheets of PV modules.
+    The six values returned by this function can be used by
+    pvsystem.calcparams_desoto to calculate the values at different
+    irradiance and cell temperature.
+
+    Parameters
+    ----------
+    celltype: str, case insensitive
+        Value is one of 'monosi', 'multisi', 'polysi' or'gaas'.
+        Others like 'cis', 'cigs', 'cdte', 'amorphous' are not implemented yet
+    v_mp: numeric
+        Module voltage at the maximum-power point at std conditions in V.
+    i_mp: numeric
+        Module current at the maximum-power point at std conditions in A.
+    v_oc: numeric
+        Open-circuit voltage at std conditions in V.
+    i_sc: numeric
+        Short-circuit current at std conditions in A.
+    alpha_sc: numeric
+        The short-circuit current (i_sc) temperature coefficient of the
+        module in units of %/K. It is converted in A/K for the computing
+        process.
+    beta_voc: numeric
+        The open-circuit voltage (v_oc) temperature coefficient of the
+        module in units of %/K. It is converted in V/K for the computing
+        process.
+    cells_in_series: numeric
+        Number of cell in the module.
+        Optional input, but helps to insure the convergence of the computing.
+    temp_ref: numeric, default 25
+        Reference temperature condition [C]
+    irrad_ref: numeric, default 1000
+        Reference irradiance condition [W/m2]
+
+    Returns
+    -------
+    Dictionnary with the following elements:
+
+    * 'I_L_ref': numeric
```

Comment:
Remove the 's from the parameters.  Have you checked how the docstring renders for readthedocs?  You have to go to readthedocs.org, sign in, and link to your pvlib-python project. You should then see a list of versions that has this branch. Click the version (branch) and it should take you to a readthedocs with your PR included.